packages/uui-ref/lib/uui-ref.element.ts

```diff
@@ -196,17 +196,21 @@ export class UUIRefElement extends SelectOnlyMixin(
         border-radius: var(--uui-border-radius);
       }
 
+      slot[name='actions']::slotted(*:first-child) {
+        --uui-button-height: calc(var(--uui-size-2) * 4);
```

Comment:
I think this one particular should be applied to all slotted element. So move it to a different query targeting ::slotted(*)

otherwise it looks good :-)